.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text eol=lf
+* text eol=lf
```

Comment:
Why do you add this line two times? isn't this redundant?

Reply:
seems wrong information from SO (first line for checkout, second for checkin). I am read now manual page and you right. Nothing about double lines.